recipes/rdkit/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "rdkit" %}
+{% set version = "Release_2017_09" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/rdkit/rdkit/archive/{{ version }}.tar.gz
+  sha256: ab83bb4d3c5986d22da0ce28d352e5f911830b63f18fee85537cebfa3df28e82
+  patches:
+    - rdpaths.patch
+    - rdconfig.patch  # [win]
+
+build:
+  skip: true  # [win and py3k]
+  skip: true  # [osx and (py27 or py35)]
+  number: 0
+
+requirements:
+  build:
+    - toolchain
+    - cmake
+    - requests
+    - boost <=1.64
+    - python
+    - numpy 1.11.*
+    - pillow
+    - freetype
+    - eigen
+    - pandas
+    - cairo
+    - pkg-config  # [not win]
+  run:
+    - cairo
+    - boost <=1.64
+    - python
+    - six
+    - numpy >=1.11
```

Comment:
Please look at these [pins]( https://github.com/conda-forge/conda-forge.github.io/blob/6bc413f9de7957194afdc239e1f89ba32b549da8/scripts/pin_the_slow_way.py#L40-L104 ) and update all of these accordingly.